lib/pstore.rb

```diff
@@ -179,137 +393,112 @@ def fetch(name, default=PStore::Error)
     end
     @table[name]
   end
+
+  # :call-seq:
+  #   pstore[key] = value
   #
-  # Stores an individual Ruby object or a hierarchy of Ruby objects in the data
-  # store file under the root _name_.  Assigning to a _name_ already in the data
-  # store clobbers the old data.
-  #
-  # == Example:
-  #
-  #  require "pstore"
+  # Creates or replaces an object or hierarchy of objects
+  # at the root for +key+:
   #
-  #  store = PStore.new("data_file.pstore")
-  #  store.transaction do  # begin transaction
-  #    # load some data into the store...
-  #    store[:single_object] = "My data..."
-  #    store[:obj_hierarchy] = { "Kev Jackson" => ["rational.rb", "pstore.rb"],
-  #                              "James Gray"  => ["erb.rb", "pstore.rb"] }
-  #  end                   # commit changes to data store file
+  #   store = PStore.new('t.store')
```

Comment:
Why does this example create a new PStore while the other examples don't?

Reply:
Fixed.